modules/ROOT/pages/database-administration/standard-databases/listing-databases.adoc

```diff
@@ -82,7 +82,7 @@ The default for a standalone database is `neo4j://localhost:7687`. label:default
 | role
 | The current role of the database (`primary`, `secondary`, `unknown`). label:default-output[]
 
-The value for composite databases is `NULL` because it does not apply to them.
+The value for composite databases is `NULL` because it does not apply to them.footnote:compositeDb[This change applies to versions 2025.04 and later and 5.25.5 and later.]
```

Comment:
still don't think 5.25 is correct, the 5.LTS is 5.26

and should we mention the 5.X version here anyway or only in the 5.x version of the docs? (and then not mention the 2025.04 in the 5.x docs)